backend/services/session.js

```diff
@@ -37,7 +37,10 @@ module.exports.session = function(config) {
 			path: "/",
 			secure: config.using_ssl
 		},
-		store: new FileStore()
+		store: new MongoDBStore({
```

Comment:
Would it be better to move MongoDBStore inside `handler/db.js` ?

Then you don't need to expose getDBUri() and MongoDBStore here.

I feel like those info are a lot closer to the dbHandler than sessions and thus it should stay overthere, with a function to get a DBStore.

Reply:
The mongodbstore is a package for using mongo as a storage for sessions it doesnt serve other purpose and wouldnt be used in any other different place ,  so I think is closer to sessions.

Reply:
Moved out to db.js